securedrop-workstation-config/debian/changelog-buster

```diff
@@ -1,3 +1,9 @@
+securedrop-workstation-config (0.1.1+buster) unstable; urgency=medium
```

Comment:
Versions don't match: comment says `0.1.2`, but the string in the changelog still says `0.1.1+buster`. When I run `PKG_VERSION=0.1.2 scripts/update-changelog securedrop-workstation-config` locally, I see the changelog version properly incremented. 

Not appending a commit to clean up, since other change log items (i.e. `-trusty`) should be removed before merge.